src/output.jl

```diff
@@ -56,6 +56,29 @@ function record_surface_speed(u, v, Nz, times, folder; colorrange = (0, 0.5), co
     end
 end
 
+function record_bottom_oxygen(O₂, times, bottom_z, folder; colorrange = (-1, 350), colormap = :turbo)
```

Comment:
Maybe make for any tracer, not only oxygen?

Reply:
Done.